packages/clerk-js/package.json

```
@@ -63,8 +63,8 @@
     "@coinbase/wallet-sdk": "4.3.0",
     "@emotion/cache": "11.11.0",
     "@emotion/react": "11.11.1",
-    "@floating-ui/react": "0.27.12",
-    "@floating-ui/react-dom": "^2.1.3",
+    "@floating-ui/react": "0.27.6",
+    "@floating-ui/react-dom": "^2.1.2",
```

Maybe remove the `^`? 🤔 
```suggestion
    "@floating-ui/react-dom": "2.1.2",
```